cmd/stringer/golden_test.go

```diff
@@ -296,11 +327,222 @@ func (i Token) String() string {
 }
 `
 
+// Simple test: enumeration of type int starting at 0.
+const month_in = `type Month int
+const (
+	January Month = iota
+	February
+	March
+	April
+	May
+	June
+	July
+	August
+	September
+	October
+	November
+	December
+)
+`
+
+const month_out = `
```

Comment:
warning: don't use underscores in Go names; const month_out should be monthOut (golint)